src/Symfony/Bundle/FrameworkBundle/Resources/config/mailer.xml

```diff
@@ -25,5 +25,11 @@
             <argument type="service" id="mailer.default_transport" />
             <tag name="messenger.message_handler" />
         </service>
+
+        <service id="mailer.envelope_listener" class="Symfony\Component\Mailer\EventListener\EnvelopeListener">
+            <argument>null</argument>
```

Comment:
We try to document when the argument is going to be injected by the extension: `<argument /> <!-- sender -->`

Reply:
Much better like that indeed. Changed it.